ArduCopter/sensors.cpp

```diff
@@ -460,10 +464,9 @@ void Copter::update_sensor_status_flags(void)
 
 #if RANGEFINDER_ENABLED == ENABLED
     if (rangefinder_state.enabled) {
-        control_sensors_present |= MAV_SYS_STATUS_SENSOR_LASER_POSITION;
         control_sensors_enabled |= MAV_SYS_STATUS_SENSOR_LASER_POSITION;
```

Comment:
This will already be there (which is wrong in case it isn't enabled), look at line 350

Reply:
Should be fixed now.